repository/volunteer_unavailability_v2.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from repository.unavailability_repository import *
+
+
+def edit_event(session, userId, eventId, title=None, start=None, end=None, periodicity=None):
+    try:
+        event = session.query(UnavailabilityTime).filter(UnavailabilityTime.eventId == eventId,
+                                                         UnavailabilityTime.userId == userId).first()
+        if event is None:
+            return False
+        if title is not None:
+            event.title = title
+        if start is not None:
+            event.start = start
+        if end is not None:
+            event.end = end
+        if end is not None:
+            event.periodicity = periodicity
+        session.commit()
+        return True
+    except Exception as e:
```

Comment:
It's good enough (I'll approve your PR), but I generally prefer to propagate exceptions up to the presentation layer (in this case the endpoint) and handle it there. It just offers the developer more clarity and flexibility. As such, I would have caught it, rolled back, and then thrown `e`